pkgs/tools/misc/diffoscope/default.nix

```diff
@@ -54,8 +55,9 @@ python3Packages.buildPythonApplication rec {
       python-magic progressbar33 pypdf2 tlsh
     ])
     ++ lib.optionals stdenv.isLinux [ python3Packages.pyxattr python3Packages.rpm acl cdrkit dtc ]
+    ++ lib.optionals (enableBloat && enableUnfree) [ apktool ]
     ++ lib.optionals enableBloat ([
-      abootimg apksigcopier apksigner apktool cbfstool colord enjarify ffmpeg fpc ghc ghostscriptX giflib gnupg gnumeric
+      abootimg apksigcopier apksigner cbfstool colord enjarify ffmpeg fpc ghc ghostscriptX giflib gnupg gnumeric
       hdf5 imagemagick libcaca llvm jdk mono ocaml odt2txt oggvideotools openssh pdftk poppler_utils procyon qemu R tcpdump ubootTools wabt radare2 xmlbeans
     ] ++ (with python3Packages; [ androguard binwalk guestfs h5py pdfminer-six ]));
```

Comment:
```suggestion
    ] ++ (with python3Packages; [ androguard binwalk guestfs h5py pdfminer-six ])
    ++ lib.optionals enableUnfree [ apktool ]);
```

Reply:
Done, let me know if I have done it wrong